tests/e2e/pact/broker/verify.ts

```diff
@@ -4,14 +4,14 @@ let endpoint: string
 
 let token: string
 
-let sleepMs: number = 0
+let sleepMs = 0
```

Comment:
Why are we removing the type from here?

Reply:
eslint done it

Reply:
Presumably because we weren't telling it anything more than it could already infer